front/pages/api/registry/[type]/lookup.ts

```diff
@@ -110,7 +109,7 @@ async function handler(
               sId: workspaceId,
             },
           });
-          if (!owner || dustWorkspaceId !== owner.sId) {
```

Comment:
I'm still not very clear with the value of dustWorkspaceId , but here it's just not used at all anymore, so I suspect we might miss something ?

Reply:
Forgot to push the latest changes 🙈. See the updated PR's description.